flytekit/remote/remote.py

```diff
@@ -1185,17 +1186,21 @@ def upload_file(
         :param project: Project to upload under, if not supplied will use the remote's default
         :param domain: Domain to upload under, if not specified will use the remote's default
         :param filename_root: If provided will be used as the root of the filename. If not, Admin will use a hash
+        :param file_name: If provided will be used as the filename.
```

Comment:
Should we still add this parameter given that we are doing the renaming in `to_pickle`?

Reply:
Not necessary. Removed it. We can add it once we need it.